package_install.sh

```diff
@@ -57,3 +57,6 @@ pushd "/usr/local/share/qemu/"
 wget ${seabios_link} -O ${seabios_file} -q
 [ -f ${seabios_file} ] || fail "can't download seabios file"
 popd
+
+cp -rap /usr/local/infrasim/template/* ~/.infrasim/template
```

Comment:
Suggest eliminating copying these templates if it is not necessary. At least doing so, there're 2 copies of template files in system.